src/TrixiTest.jl

```diff
@@ -1,6 +1,7 @@
 module TrixiTest
 
-using Test: @test, @testset
+using Test: @inferred, @test, @test_broken, @test_deprecated, @test_logs,
+            @test_nowarn, @test_skip, @test_throws, @test_warn, @testset
```

Comment:
It feels a bit strange to import all of these macros explicitly even though they are not needed here. I know we somehow need this in order to access them from the tests in Trixi.jl, but would this also work with a simple `using Test`?